### ISSUE TITLE
systemPreferences.getColors() throws exceptions with some macOS colors

### ISSUE THREAD

**Issue by yourhead** [electron/electron#20599]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 
v7.0.0-beta.6
* **Operating System:**
  * 
macOS 10.14.6
* **Last Known Working Electron version:**
  * 
the bug exists in all recent betas

### Expected Behavior

a color value as a string
no exception thrown

### Actual Behavior

exception thrown
```
"Error: Unknown color: disabled-control-text
    at Function.updateAppearanceTheme (/Users/iac/Desktop/gluon/dist/App.js:138:54)
    at App.createWindow (/Users/iac/Desktop/gluon/dist/App.js:9:13)
    at App.emit (events.js:208:15)"
```

### To Reproduce

```
systemPreferences.getColor('disabled-control-text');
systemPreferences.getColor('scrubber-textured-background');
systemPreferences.getColor('selected-menu-item');
```

For reference, here are the macOS headers for these colors and their availability in the macOS 10.14 SDK

```
@property (class, strong, readonly) NSColor *disabledControlTextColor;                     // Text on disabled controls

@property (class, strong, readonly) NSColor *scrubberTexturedBackgroundColor NS_AVAILABLE_MAC(10_12_2); // Patterned background color for use in NSScrubber

@property (class, strong, readonly) NSColor *selectedMenuItemColor NS_DEPRECATED_MAC(10_0, API_TO_BE_DEPRECATED, "Use NSVisualEffectMaterialSelection");
```

All three are available in macOS 10.14 (one is deprecated, but still available).

But even if they weren't, the Electron docs don't specify limitations on the macOS version for this function. Shouldn't I expect *some* value rather than throwing an exception even when colors are not available on the user's system.

Or perhaps the documentation should reflect that in some cases the function will throw an exception -- and what those cases are.

A fiddle for such basic stuff seems overkill, but it also took me about 1 minute -- so why not. 😃 

https://gist.github.com/yourhead/1db43ba064651648c673d686775b7afc

**Comment by MarshallOfSound** [comment]
If I'm reading this right we've documented that as a color we support but the code has no reference to it.

https://github.com/electron/electron/blob/master/shell/browser/api/atom_api_system_preferences_mac.mm#L506

cc @codebytere any idea what's going on here?

**Comment by yourhead** [comment]
Kindof -- the three have identical bug symptoms, but the error mechanism is different in each case.

 - `disabled-control-text`: this is simply a mistyped string. The Obj-C call is made correctly to `[NSColor disabledControlTextColor];` on line 518. but the API key is mistyped as `disabled-control` omitting `text`. However the documentation shows it correctly as does the typescript type (which is how I stumbled upon the problem).

- `selected-menu-item`: this one is simply a missing case. selected-menu-item-text is there as it should be, but the item itself (the background menu color i believe) is missing.

- `scrubber-textured-background-color`: i was not able to identify an error mechanism by inspection. the entry exists on line 538. The Obj-C call is correct. The `@available` is correct. `¯\_(ツ)_/¯` -- that one is for you folks.  😝 


Are there unit tests for these? Could you point me towards that code? It would help me identify things in future reports. 😃 


Also worth mentioning:  Electron throws exceptions on unavailable colors.

e.g. line 519 of `atom_api_system_preferences_mac`:

```
  } else if (color == "find-highlight") {
    if (@available(macOS 10.14, *))
      sysColor = [NSColor findHighlightColor];
```

This means that `systemPreferences.getColor('find-highlight-color');` will throw an exception when the app is running on macOS 10.13, but not on macOS 10.14. But that's not documented and might come as a surprise for those unfamiliar with the macOS SDK. Other macOS requirements *are* documented. This gives the impression that the `getColor` API polyfills where macOS doesn't supply colors.

I suspect this later bit is a documentation bug and maybe should be filed separately. But I thought I'd mention it here just in case.  😃

**Comment by codebytere** [comment]
I'll handle this as soon as i can!
